pyseir/inference/model_fitter.py

```diff
@@ -240,36 +240,47 @@ def set_inference_parameters(self):
             self.fit_params["hosp_fraction"] = 1
 
         if len(self.fips) == 5:
-            OBSERVED_NEW_CASES_GUESS_THRESHOLD = 2
-            idx_enough_cases = np.argwhere(
-                np.cumsum(self.observed_new_cases) >= OBSERVED_NEW_CASES_GUESS_THRESHOLD
-            )[0][0]
-            initial_cases_guess = np.cumsum(self.observed_new_cases)[idx_enough_cases]
-            t0_guess = list(self.times)[idx_enough_cases]
-
             state_fit_result = load_inference_result(fips=self.state_obj.fips)
-            self.fit_params["t0"] = t0_guess
+            T0_LEFT_PAD = 5
+            T0_RIGHT_PAD = 30
+
+            # See if we can do better based on the actual data
+
+            # Let's look at the time when we've got a fifth reported case in a fips
```

Comment:
I don't see any references to requiring the fifth reported case. What am I missing here?

Reply:
Fixed. I wanted to make it the 5th value, but thought better to not change multiple things at once.